plugins/inputs/squid/README.md

```diff
@@ -0,0 +1,104 @@
+# Squid Input Plugin
+
+This plugin gathers metrics from a Squid web proxy cache server (http://www.squid-cache.org/).
+
+### Configuration:
+
+To use this plugin you must [configure the manager api to allow access](https://wiki.squid-cache.org/Features/CacheManager) from the system on which telegraf is running
+
+```toml
+# Squid web proxy cache plugin
+[[inputs.squid]]
+  ## url of the squid proxy manager counters page
+  url = http://localhost:3128/squid-internal-mgr/counters
```

Comment:
put the url in quotes. and just take the `scheme://host:port`; append the `/squid-internal-mgr/counters` in the code